src/helpers/mathjax.ts

```diff
@@ -0,0 +1,123 @@
+import { Element } from '@openstax/types/lib.dom';
+import debounce from 'lodash/debounce';
+import isEmpty from 'lodash/isEmpty';
+import memoize from 'lodash/memoize';
+import WeakMap from 'weak-map';
+
+const MATH_MARKER_BLOCK  = '\u200c\u200c\u200c'; // zero-width non-joiner
```

Comment:
Was this copied from tutor? Just curious.

Reply:
yea this whole file was converted from https://github.com/openstax/tutor-js/blob/master/shared/src/helpers/mathjax.js